### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by pasternakewa**

### CODE REVIEW

File: src/components/About.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+import recycle from './assets/recycle.jpg'
+import palette from './assets/palette.jpg'
 
 export default function About() {
   return (
-    <div>
-      <h1>About Makeup Showroom</h1>
+    <div className="about-container">
+      <h1 style={{marginBottom: '2rem'}}>About Makeup Showroom</h1>
+      <div className="about-section">
```
Comment (vdanylov) [inline src/components/About.js]: @pasternakewa if You have repeated partes, I prefer just create an object and iterate through it

File: src/components/GalleryImage.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function GalleryImage({ category, id, name, src }) {
+  return (
```
Comment (vdanylov) [inline src/components/GalleryImage.js]: @pasternakewa could be removed

File: src/components/Products.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ export default function Products() {
       <div className="gallery">
         {isLoading && <h4>Loading products...</h4>}
         {getUniqueCategoryProducts.map((item) => (
```
Comment (vdanylov) [inline src/components/Products.js]: @pasternakewa possible error `getUniqueCategoryProducts` is it always exist?

File: src/components/GalleryImage.js (hunk 2 above)
Comment (pasternakewa) [reply src/components/GalleryImage.js]: @vdanylov what exactly?

File: src/components/About.js (hunk 1 above)
Comment (pasternakewa) [reply src/components/About.js]: > done

File: src/components/Products.js (hunk 3 above)
Comment (pasternakewa) [reply src/components/Products.js]: * do
  * ne

File: src/components/Image.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
-import React from "react";
+import React from 'react'
```
Comment (vdanylov) [inline src/components/Image.js]: @pasternakewa You ahve strange formatting sometimes single quote sometimes double. I think You formatter is off

File: src/components/Products.js [hunk 5]
```diff
@@ -1,9 +1,11 @@
 import React, { useState, useEffect } from "react";
-import Image from "./Image";
+import GalleryImage from "./GalleryImage";
+import { Link } from "react-router-dom";
 
 export default function Products() {
   const [items, setItems] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(null)
```
Comment (vdanylov) [inline src/components/Products.js]: @pasternakewa   const [isError, setIsError] = useState(null) ->   const [isError, setIsError] = useState()

File: src/components/GalleryImage.js (hunk 2 above)
Comment (vdanylov) [reply src/components/GalleryImage.js]: @pasternakewa return, just create arrow function

File: src/components/Products.js (hunk 3 above)
Comment (vdanylov) [reply src/components/Products.js]: wot?

File: src/App.css [hunk 6]
```diff
@@ -78,6 +78,8 @@ main {
   font-size: 20px;
 }
 
+/* ___Products___ */
```
Comment (vdanylov) [inline src/App.css]: @pasternakewa Remove

File: src/components/Products.js [hunk 7]
```diff
@@ -13,28 +15,33 @@ export default function Products() {
...
 
   let getUniqueCategoryProducts = [
-    ...new Map(items.map((item) => [item.product_type, item])).values()
+    ...new Map(items?.map((item) => [item.product_type, item])).values(),
   ];
-
   return (
     <div className="gallery-container">
       <h1 className="gallery-title">Products</h1>
+      {(isError || (!isLoading && items.length === 0)) && (
```
Comment (vdanylov) [inline src/components/Products.js]: @pasternakewa please move it to separate variable 
`const showSmth =  isError || (!isLoading && items.length === 0)`

File: src/components/Products.js [hunk 8]
```diff
@@ -13,28 +15,33 @@ export default function Products() {
...
+        <p>
+          Problem loading data. {isError} Get back to{" "}
+          <Link to="/">home page</Link>.
+        </p>
+      )}
       <div className="gallery">
         {isLoading && <h4>Loading products...</h4>}
-        {getUniqueCategoryProducts.map((item) => (
-          <Image
+        {getUniqueCategoryProducts?.map((item) => (
```
Comment (vdanylov) [inline src/components/Products.js]: @pasternakewa `!isLoading && getUniqueCategoryProducts  ? getUniqueCategoryProducts.map() : null`

Comment (vdanylov) [inline src/components/Products.js]: @pasternakewa please use these vars interchangeably

File: src/components/Image.js [hunk 9]
```diff
@@ -1,10 +1,11 @@
 import React from "react";
 
-export default function Image({ category, id, name, src }) {
+const Image = ({ src, name, className }) => {
   return (
```
Comment (vdanylov) [inline src/components/Image.js]: @pasternakewa remove

File: src/components/Footer.js [hunk 10]
```diff
@@ -1,35 +1,55 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-
 export default function Footer() {
   return (
     <div className="footer-container">
       <footer>
```
Comment (vdanylov) [inline src/components/Footer.js]: @pasternakewa remove return

**Pull request merged**